examples/simple-game-server/main.go

```diff
@@ -360,6 +360,69 @@ func handleResponse(txt string, s *sdk.SDK, cancel context.CancelFunc) (response
 		}
 		response, responseError = setCounterCapacity(s, parts[1], parts[2])
 		addACK = false
+
+	case "GET_LIST_CAPACITY":
```

Comment:
Not for this PR, more thinking aloud -

I wonder if this has got big enough that it'd be worth refactoring this into something a bit less verbose.

Maybe a map of command values (rather than a switch) that get registered at startup with some kind of callback function?

Reply:
Good point, it is getting to be quite a long method.